src/components/WebexMeetingControl/WebexMeetingControl.jsx

```diff
@@ -68,19 +68,31 @@ function renderButton(action, display, style, showText, asItem) {
  * @param {Function} action  Adapter control callback
  * @param {object} display  Display data of the control
  * @param {object} style  Custom style to apply
+ * @param {string} type  Name of the control as defined in adapter
  * @returns {object} JSX of the component
  */
-function renderDropdown(action, display, style) {
-  const {options, noOptionsMessage, selected} = display;
+function renderDropdown(action, display, style, type) {
+  const {noOptionsMessage} = display;
+  let {options, selected} = display;
+  // The browser api setSinkId() does not work properly on Firefox and Safari and that's why we need to overwrite/reset received options on the control.
```

Comment:
// The browser api setSinkId() does not work properly on Firefox and Safari browsers so we need to treat them separately by resetting the options we receive with a default no media label.